src/hotspot/os/linux/cgroupV1Subsystem_linux.cpp

```diff
@@ -266,3 +275,17 @@ jlong CgroupV1Subsystem::pids_max() {
   char * pidsmax_str = pids_max_val();
   return limit_from_str(pidsmax_str);
 }
+
+/* pids_current
+ *
+ * The number of tasks currently in the cgroup (and its descendants) of the process
+ *
+ * return:
+ *    current number of tasks
+ *    OSCONTAINER_ERROR for not supported
+ */
+jlong CgroupV1Subsystem::pids_current() {
```

Comment:
`pids.current` never contains a string `max` (for unlimited). Therefore, we shouldn't need to do the `pids_current_val` -> `limit_from_str()` trick. We should be able to use `GET_CONTAINER_INFO(int, [...]` akin to `cpu_quota`. `int` or `long` would both be suitable. Up to you to decide which data type to use. I don't think it'll ever be beyond the maximum integrer value.